tests/integration/targets/meraki_mr_radio/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# Test code for the Meraki modules
+# Copyright: (c) 2021, Tyler Christiansen (@supertylerc)
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- block:
+  - name: Create network with type wireless
+    meraki_network:
+      auth_key: "{{ auth_key }}"
+      state: present
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      type: wireless
+      timezone: America/Chicago
+    delegate_to: localhost
+
+  - name: Add access points to network
+    meraki_device:
+      auth_key: "{{ auth_key }}"
+      state: present
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      serial: "{{ serial_wireless }}"
+    delegate_to: localhost
+
+  - name: Create RF profile
+    meraki_mr_rf_profile:
+      auth_key: "{{ auth_key }}"
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      state: present
+      name: Test Profile
+      band_selection_type: ap
+      client_balancing_enabled: True
+      ap_band_settings:
+          mode: dual
+          band_steering_enabled: true
+      five_ghz_settings:
+        max_power: 10
+        min_bitrate: 12
+        min_power: 8
+        rxsop: -65
+        channel_width: 20
+        valid_auto_channels:
+          - 36
+          - 40
+          - 44
+      two_four_ghz_settings:
+        max_power: 10
+        min_bitrate: 12
+        min_power: 8
+        rxsop: -65
+        ax_enabled: false
+        valid_auto_channels:
+          - 1
+    delegate_to: localhost
+    register: create
+
+  - set_fact:
+      profile_id: "{{ create.data.id }}"
+
+  - name: Configure AP radios in check mode
+    meraki_mr_radio:
```

Comment:
I'll test this more but the integration test in check mode is hanging. Have you seen that problem?

```
TASK [meraki_mr_radio : Add access points to network] *******************************************************
changed: [localhost]

TASK [meraki_mr_radio : Create RF profile] ******************************************************************
changed: [localhost]

TASK [meraki_mr_radio : set_fact] ***************************************************************************
ok: [localhost]

TASK [meraki_mr_radio : Configure AP radios in check mode] **************************************************
```

I've been waiting multiple minutes.

Reply:
I haven't seen this previously and have used check mode during testing quite a bit.  Not sure where/why that's hanging.  Do you get anything from verbose output or when it (eventually) fails?